util/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -23,7 +23,7 @@ org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.autoboxing=ignore
 org.eclipse.jdt.core.compiler.problem.comparingIdentical=error
 org.eclipse.jdt.core.compiler.problem.deadCode=warning
-org.eclipse.jdt.core.compiler.problem.deprecation=error
+org.eclipse.jdt.core.compiler.problem.deprecation=disabled
```

Comment:
Here, somehow ECJ is not getting the `release` parameter we pass to the Java compiler, so it is reporting warnings for methods deprecated after JDK 11.  Maybe there is a way to tell ECJ which Java version we are targeting, and then it won't report such errors?

Reply:
I don't know of a way to do that offhand, but I haven't actively investigated. If you think there's a decent chance that I'll find something you missed, feel free to create a new issue as a reminder and assign it to me.

Reply:
I think this one is just safe to leave as is.  We already compile `util` with `-Werror` so any true deprecation issues should cause `javac` itself to fail.